composer.json

```diff
@@ -13,17 +13,17 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "symfony/browser-kit": "~2.0|~3.0",
-        "symfony/config": "~2.0|~3.0",
-        "symfony/dependency-injection": "~2.0|~3.0",
-        "symfony/form": "~2.0|~3.0",
-        "symfony/http-foundation": "~2.0|~3.0",
-        "symfony/http-kernel": "~2.0|~3.0",
+        "symfony/browser-kit": "~2.0|~3.0|~4.0",
+        "symfony/config": "~2.0|~3.0|~4.0",
+        "symfony/dependency-injection": "~2.0|~3.0|~4.0",
+        "symfony/form": "~2.0|~3.0|~4.0",
+        "symfony/http-foundation": "~2.0|~3.0|~4.0",
+        "symfony/http-kernel": "~2.0|~3.0|~4.0",
```

Comment:
I think we can safely bump to `~2.7|~3.0|~4.0` to only support, still-supported Symfony versions

Reply:
Sounds reasonable, PHP min version now also moved up to 5.3.9 (enforced by Symfony 2.7)